src/lib/components/Swap/Summary/index.tsx

```diff
@@ -59,7 +59,7 @@ const Body = styled(Column)<{ open: boolean }>`
     }
 
     ${Estimate} {
-      max-height: ${({ open }) => (open ? 0 : 56 / 12)}em; // 2 * line-height + padding
+      max-height: ${({ open }) => (open ? 0 : 6)}em; // 2 * line-height + padding
```

Comment:
This was set to 56/12 for the animation to be smooth - did you visually test this?

Reply:
bump

Reply:
Had to change when i was rendering more text - but now less text rendered so reverting